### PR TITLE
[CU-2x7dbfp] Move parachain files to own directory

### PR DESCRIPTION
**Pull request by andor0**
## Issue
https://app.clickup.com/t/2x7dbfp

## Description
1. moved all rust code to `code` directory:
```bash
mkdir -p code/parachain
git mv {Cargo.lock,Cargo.toml,clippy.toml,integration-tests,rustfmt.toml,simnode,utils,xcvm} code/
git mv {frame,node,runtime,src} code/parachain/
```
2. updated CI configs and flake.nix
3. updated paths to pallets in mdbook
4. updated cspell.yaml
5. updated dictionaries of `cspell` and renamed variables
6. updated CODEOWNERS
7. updated paths in Makefile, Makefile.toml, and scripts

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2x7dbfp Move parachain files to own directory](https://app.clickup.com/t/2x7dbfp)

**Comment by andor0** [comment]
@Mergify refresh

**Review by haroldsphinx — APPROVED** [review]

File: flake.nix [hunk 1]
```diff
@@ -181,19 +158,31 @@
...
+                  type == "regular" && lib.strings.hasSuffix ".nix" name;
                 customFilter = name: type:
-                  !((type == "directory"
-                    && builtins.elem (baseNameOf name) directory-blacklist)
-                    || (type == "regular"
-                      && builtins.elem (baseNameOf name) file-blacklist)
+                  !((isBlacklisted name type) || (isImageFile name type)
+                    || (isPlantUmlFile name type)
                     # assumption that nix is final builder, 
                     # so there would no be sandwich like  .*.nix <- build.rs <- *.nix
```
Comment (dzmitry-lahoda) [inline flake.nix:180]: this commen was reason for .nix negative filter .

**Review by KaiserKarel — APPROVED** [review]

**Review by dzmitry-lahoda — APPROVED** [review]

File: .config/dictionaries/terminology.dic [hunk 2]
```diff
@@ -16,6 +16,8 @@ suri # Substrate Uri
 parathread
 parathreads
 CurrencyId
+extrinisic
+extrinisics
```
Comment (Martin-Kiss) [inline .config/dictionaries/terminology.dic:20]: Extrinsics is already in the dictionary and these contain typos.
```suggestion
```

**Review by Dom-Roth — CHANGES_REQUESTED** [review]

**Review by Martin-Kiss — CHANGES_REQUESTED** [review]

**Review by benluelo — APPROVED** [review]

File: .config/dictionaries/terminology.dic (hunk 2 above)
[the remark this answers is not in the document]
Comment (andor0) [reply .config/dictionaries/terminology.dic]: fixed

File: code/parachain/runtime/common/src/impls.rs [hunk 3]
```diff
@@ -16,9 +16,9 @@ where
 	<R as balances::Config>::Balance: From<u128>,
 {
 	fn on_nonzero_unbalanced(amount: NegativeImbalance<R>) {
-		/// TODO (vim): This configuration must be a part of each runtime configuration instead
-		/// being a  common configuration. We could build a small custom pallet to capture this
-		/// configuration while  also making part of on-chain governance through extrinisics.
+		// TODO (vim): This configuration must be a part of each runtime configuration instead
+		// being a  common configuration. We could build a small custom pallet to capture this
+		// configuration while  also making part of on-chain governance through extrinisics.
```
[the remark this answers is not in the document]
Comment (andor0) [reply code/parachain/runtime/common/src/impls.rs]: fixed

File: .config/dictionaries/terminology.dic (hunk 2 above)
Comment (andor0) [reply .config/dictionaries/terminology.dic]: fixed

**Review by Martin-Kiss — APPROVED** [review]

**Comment by andor0** [comment]
@mergify refresh

**Pull request merged**